src/microceph.py

```diff
@@ -46,6 +50,48 @@ def _run_cmd(cmd: list) -> str:
         raise e
 
 
+def is_ready() -> bool:
```

Comment:
The charm has a .ready_for_service() method too that seems to do the same, maybe could de-duplicate

Reply:
refactored, ready_for_service to use microceph.is_ready internally.